This is an instruction for part 2 of the GWAS tutorial (no coding)

1. Choose a gene of interest (ideally related to your research area or a phenotype you care about). Briefly state why you chose this gene (1â€“2 sentences for context).
2. Start a powerpoint/google slides presentation on your gene, make a title slide with the gene symbol
3. Look at it in gnomAD. Is it highly constrained? How many loss of function variants? How many missense variants? - this will be slide (1)
4. Retrieve [GWAS Catalog](https://www.ebi.ac.uk/gwas/) associations for variants within or near your gene: What traits are associated? Nottice effect sizes and p-values. Are associations consistent across studies? Are they biologically plausible? - slide (2)
5. Check [genebass](put link here). Any associations with loss of function/missense/synonymous variants? - find your favourite one. Compare rare variant associations to GWAS findings Do rare and common variants implicate the same biology? slide (3)
6. Finally, check [Enrichr](https://maayanlab.cloud/Enrichr/) - Which pathways is your gene involved in? Diseases? Anything surprising? - slide (4)
7. Make a one-sentence summary with a conclusion from your research, preferably something you did not know before. Maybe state one hypothesis you would test in a follow-up study. - slide (5)

Bonus - look at the [GTEx Portal](https://gtexportal.org/home/). Which tissues is your gene expressed in?
